sdk/containerregistry/azure-containerregistry/azure/containerregistry/_anonymous_exchange_client.py

```diff
@@ -23,7 +23,7 @@ class AnonymousACRExchangeClient(object):
     """
 
     def __init__(self, endpoint, **kwargs):  # pylint: disable=missing-client-constructor-parameter-credential
-        # type: (str, Dict[str, Any]) -> None
+        # type: (str, Any) -> None
```

Comment:
Can we use Python3 type hints?

Reply:
To make this PR easier to review, I didn't include the hints change. I'll create a separate for it.